utilities/rpp-unittests/HOST_NEW/testAllScript.sh

```diff
@@ -154,15 +171,67 @@ rm -rvf ./*
 cmake ..
 make -j16
 
+USER_INPUT_ARGS=("")
+CASE_INDEX=0
+if ([[ $USER_FLAG -eq "1" ]])
+then
+    for ((case=$CASE_START;case<=$CASE_END;case++))
+    do
+        if [[ case -eq 0 ]]
+        then
+            varNames="Brightness - Alpha, Beta: "
+        elif [[ case -eq 1 ]]
+        then
+            varNames="Gamma correction - Gamma: "
+        elif [[ case -eq 2 ]]
+        then
+            varNames="Blend - Alpha: "
+        elif [[ case -eq 13 ]]
+        then
+            varNames="Exposure - Exposure Factor: "
+        elif [[ case -eq 31 ]]
+        then
+            varNames="ColorCast - R, G, B, Alpha: "
+        elif [[ case -eq 36 ]]
+        then
+            varNames="ColorTwist - Brightness, Contrast, Hue, Saturation: "
+        elif [[ case -eq 37 ]]
+        then
+            varNames="Crop - x, y, w, h: "
+        elif [[ case -eq 38 ]]
+        then
+            varNames="CropMirrorNormalize - x, y, w, h, Mean, StdDev, Mirror: "
+        elif [[ case -eq 81 ]]
+        then
+            varNames="ColorJitter - Brightness, Contrast, Hue, Saturation: "
+        elif [[ case -eq 83 ]]
+        then
+            varNames="GridMask - TileWidth, GridRatio, GridAngle, TranslateX, TranslateY: "
+        elif [[ case -eq 84 ]]
+        then
+            varNames="Spatter - R, G, B: "
+        fi
+        read -p "$varNames" USER_INPUT_ARGUMENTS[CASE_INDEX]
```

Comment:
Could you confirm if these work - the unit test suite should run in the scenarios:
- running single case - with user params or default values
- running say a couple cases - say 3 cases (./testAllScript 0 2 0 to run case 0, 1, 2) - with user params or default values
- And finally, the all-cases output test where we just run ./testAllScript.sh should run ideally with just the default values like it does now.